pgjdbc/src/test/java/org/postgresql/test/sspi/SSPITest.java

```diff
@@ -0,0 +1,72 @@
+package org.postgresql.test.sspi;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.fail;
+import static org.junit.Assume.assumeThat;
+
+import org.postgresql.test.TestUtil;
+import org.postgresql.util.PSQLException;
+import org.postgresql.util.PSQLState;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.sql.Connection;
+import java.sql.Statement;
+import java.util.Properties;
+
+/*
+ * These tests require a working SSPI authentication setup
```

Comment:
Well, it looks odd when a non-javadoc comment is present in a javadoc location.

It is not that I'm going to read those javadocs in html format. Having them in javadoc format would help to `{@link OtherClasses}`, keep those in sync with code when refactoring (e.g. from IDE), etc, etc.